ui/app/AppMain.qml

```diff
@@ -154,6 +154,10 @@ RowLayout {
             if(this.children[currentIndex] === browserLayoutContainer && browserLayoutContainer.active == false){
                 browserLayoutContainer.active = true;
             }
+
+            if(this.children[currentIndex] === chatLayoutContainer){
+                chatLayoutContainer.chatColumn.chatMessages.chatLogView.scrollToBottom(true);
```

Comment:
I feel like it would be cleaner if we used an `active` property like with the browser above.
that way, we could put a `Connections` in ChatMessages with a target on the `chatLayoutContainer` for `onActiveChnaged`, if `true`, `scrollToBottom`